lfroms/clouds
# Add iOS 14 home screen widget

### What is the problem being solved in this PR?
This PR adds a home screen widget that displays weather conditions at the user's current location.

### Related issues or PRs
Resolves #53 

### Why did you choose this approach?
This is just preliminary functionality, additional widget features to be added in the future.

### How to test changes
1. Install the app.
1. Edit the home screen, add the widget.  Ensure that it displays the weather.

### Checklist
